tutorials/README.md

```diff
@@ -36,12 +36,34 @@ Some cryptographic functionalities of Aleph use curve secp256k1 and require inst
 Archiving programs and volumes requires
 [Squashfs user space tools](https://github.com/plougher/squashfs-tools).
 
-    sudo apt-get install -y python3-pip libsecp256k1-dev squashfs-tools
+- Linux : 
+
+``` 
+sudo apt-get install -y python3-pip libsecp256k1-dev squashfs-tools
+``` 
+
+- macOs : 
+
+``` 
+brew tap cuber/homebrew-libsecp256k1
+brew install libsecp256k1 squashfs
+```
+
 
 You will also need [Uvicorn](https://www.uvicorn.org/) for local testing 
 and the [Python Aleph client](https://github.com/aleph-im/aleph-client) for it's command-line tools:
 
-    pip3 install uvicorn[standard] aleph-client fastapi eth_account
+- Linux : 
+ 
+``` 
+pip3 install uvicorn[standard] aleph-client fastapi eth_account
+``` 
+
+- macOs : 
```

Comment:
Can you explain why different instructions for Linux and macOS when using pip3 ?

Reply:
It works with quotes for Linux as well

Reply:
Then I do not understand why having different sections for Linux and a macOS, when they could be identical.

Reply:
should be one section but with quotes , that's the update

Reply:
These quotes also work on Linux. Can you update the document to use them for both OS ?